tests/lax_control_flow_test.py

```diff
@@ -623,6 +624,18 @@ def loss(test_params):
 
     api.grad(loss)(0.25)  # doesn't crash
 
+  def testIssue744(self):
+    Point = collections.namedtuple('Point', ['x', 'y'])
+    p0 = Point(x=np.array(1), y=np.array(2))
+
+    def plus_one(p, iter_idx):
+      return Point(p.x+1, p.y+1), iter_idx
+
+    self.assertRaisesRegexp(
+        ValueError,
+        '.*no leading.*',
```

Comment:
I like putting the full error message in this kind of test (possibly still using a regex if there's an element that changes), which forces you + your reviewer if you have one to actually look at the rendered message and see if it's actionable. This is also good for catching stupid formatting errors, but it's annoying if the error changes a lot. Just my $.02.

Reply:
Great idea! Fixed.